apps/machine/zynqmp/platform_info.c

```diff
@@ -36,28 +36,48 @@
                                * one CPU in the exmaple. We set the CPU
                                * index to 0. */
 #ifdef versal
+#ifndef IPI_CHN_BITMASK
 #define IPI_CHN_BITMASK     0x08 /* IPI channel bit mask for IPI
 					* from/to RPU0 */
+#endif /* !IPI_CHN_BITMASK */
+#ifndef IPI_DEV_NAME
 #define IPI_DEV_NAME        "ff360000.ipi" /* IPI device name */
+#endif /* !IPI_DEV_NAME */
 #else
+#ifndef IPI_CHN_BITMASK
 #define IPI_CHN_BITMASK	    0x00000100
+#endif /* !IPI_CHN_BITMASK */
+#ifndef IPI_DEV_NAME
 #define IPI_DEV_NAME	    "ff340000.ipi"
+#endif /* !IPI_DEV_NAME */
 #endif /* versal */
 #define DEV_BUS_NAME        "platform" /* device bus name. "platform" bus
                                         * is used in Linux kernel for generic
 					* devices */
 /* libmetal devices names used in the examples.
  * They are platform devices, you find them in Linux sysfs
  * sys/bus/platform/devices */
+#ifndef SHM_DEV_NAME
 #define SHM_DEV_NAME        "3ed20000.shm" /* shared device name */
-
+#endif /* SHM_DEV_NAME */
```

Comment:
Do you really need to give a name that depend on the address?
Same for POLL_DEV_NAME and IPI_DEV_NAME

Reply:
for linux a name is needed as the device is accessed via UIO and device tree node name